commands/command_untrack.go

```diff
@@ -57,7 +57,7 @@ func untrackCommand(cmd *cobra.Command, args []string) {
 
 		path := strings.Fields(line)[0]
 		if removePath(path, args) {
-			Print("Untracking %s", path)
+			Print("Untracking %s", unescapeTrackPattern(path))
```

Comment:
> I'm thinking that we should quote the pattern that comes out to make it clear what's going on with whitespace-only patterns.

I agree, even if a whitespace-only pattern is probably not that useful :) Can you change this and the "tracking" message to use `%q`?